cmd/synthetic-monitoring-agent/main.go

```diff
@@ -74,6 +75,7 @@ func run(args []string, stdout io.Writer) error {
 			AutoMemLimit         bool
 			MemLimitRatio        float64
 			DisableK6            bool
+			DisableUsageReports  bool
```

Comment:
This now defaults to false, which means the agent will collect usage reports by default.